packages/playwright-test/src/runner.ts

```diff
@@ -243,6 +243,13 @@ export class Runner {
         return true;
       const name = project.name.toLocaleLowerCase();
       unknownProjects!.delete(name);
+      for (const value of projectsToFind) {
```

Comment:
`projectsToFind` used to normalize the names for case-insensitive processing. Now that the project names are regex, you can make regex matching `i`, case-insensitive, so you should not need this `projectsToFind` normalization.